packages/react-admin-stories/src/react-admin-date-picker/DateRangePicker.tsx

```diff
@@ -0,0 +1,40 @@
+import { storiesOf } from "@storybook/react";
+import * as moment from "moment";
+import * as React from "react";
+import { DateRangePicker as AirBNBDateRangePicker } from "react-dates";
+import * as sc from "./Picker.sc";
+
+const Story = () => {
+    moment.locale("de");
+    const [focusedInputField, setFocusedInputField] = React.useState<"startDate" | "endDate" | null>(null);
+    const [pickedStartDate, setPickedStartDate] = React.useState<moment.Moment | null>(moment());
+    const [pickedEndDate, setPickedEndDate] = React.useState<moment.Moment | null>(null);
+
+    return (
+        <sc.DateRangePickerWrapper>
```

Comment:
Shouldn't the stories demonstrate the use of our own components? Otherwise adding this new package would be rather useless.